tests/migration_test.go

```diff
@@ -2405,11 +2405,16 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 										continue
 									}
 									// Wait for the iso to be created
-									Eventually(func() string {
+									Eventually(func() error {
 										output, err := tests.RunCommandOnVmiTargetPod(vmi, []string{"/bin/bash", "-c", "[[ -f " + volPath + " ]] && echo found || true"})
-										Expect(err).ToNot(HaveOccurred())
```

Comment:
The error was that the container is paused. If the command keeps error-ing (because the container never un-pauses for example), I want to be able to see that error in the test logs, that's why I return it.

Reply:
Makes sense!